client/src/components/Account/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,78 @@
+import { Box, List, ListItem, ListItemText, Typography } from '@material-ui/core';
+import { Link } from 'react-router-dom';
+import { useLocation } from 'react-router';
+
+import { useStyles } from './useStyles';
+
+export default function SideBar(): JSX.Element {
+  const classes = useStyles();
+  const { pathname } = useLocation();
+  const path = pathname.split('/')[2];
+  console.log(path === 'editprofile' ? 'classes.active' : '');
+  return (
+    <Box maxWidth={250} className={classes.sideBarContainer}>
+      <List className={classes.sideBar}>
+        <ListItem>
+          <Link to="/user/editprofile" className={`${classes.links} ${path === 'editprofile' ? classes.active : ''}`}>
+            <ListItemText
+              disableTypography
+              primary={
+                <Typography variant="h6" className={classes.linkText}>
```

Comment:
i would avoid using h6 tag here for accessibility, as header tags are reserved for headers of a page. this is rather a link so if you're utilizing it for font size, i would just change it via linkText

Reply:
Yes, the sole purpose for using the heading tag was font size. I have updated it as per guidelines :)